internal/logs/logger.go

```diff
@@ -1,8 +1,153 @@
 package logs
 
 import (
+	"fmt"
 	"log"
 	"os"
+	"strings"
 )
 
-var Logger = log.New(os.Stdout, "", log.LstdFlags)
+type Level int
+
+const (
+	DebugLevel Level = iota
+	InfoLevel
+	WarnLevel
+	ErrorLevel
+)
+
+var levelMap = map[string]Level{
+	"debug": DebugLevel,
+	"info":  InfoLevel,
+	"warn":  WarnLevel,
+	"error": ErrorLevel,
+}
+
+const (
+	colorReset  = "\033[0m"
+	colorRed    = "\033[31m"
+	colorYellow = "\033[33m"
+	colorBlue   = "\033[34m"
+	colorWhite  = "\033[37m"
+	colorCyan   = "\033[36m"
+)
```

Comment:
Adding colours to this is a nice touch for development 👍 I'm wondering if we should enable this by default tho. In production you don't really want to get coloured output. Could we disable it by default and have it enabled only in dev somehow? Other option would be to use [`NO_COLOR`](https://no-color.org/), but then it needs to be disabled by everyone. We could also apply that env var in the n8n docker image, or only at runtime

Reply:
Good idea! Since `NO_COLOR` is a standard, I added support for it and we can later add it to the docker image.